front/pages/api/v1/w/[wId]/assistant/conversations/index.ts

```diff
@@ -205,12 +205,17 @@ async function handler(
       let newMessage: UserMessageType | null = null;
 
       if (contentFragment) {
+        // hack: for users creating content_fragments through our public API
+        const contentType =
+          contentFragment.contentType === "file_attachment"
+            ? "text/plain"
+            : contentFragment.contentType;
```

Comment:
Do you mind creating a helper function that we can use in both places  + some logging 🙏